src/Microsoft.AspNetCore.Razor.Language/Legacy/ParserHelpers.cs

```diff
@@ -27,7 +27,7 @@ public static bool IsNewLine(char value)
         public static bool IsNewLine(string value)
         {
             return (value.Length == 1 && (IsNewLine(value[0]))) ||
-                   (string.Equals(value, Environment.NewLine, StringComparison.Ordinal));
```

Comment:
Whenever the document contains CRLF in VS for Mac, this check used to return false which ended up breaking few indentation scenarios.